config/settings.yml

```diff
@@ -12,6 +12,13 @@ financial:
     csv: true
     xml: true
 
+converters:
+  default:
+    product_to_journal_rows: Converters::ProductToJournalRows
+    order_detail_to_journal_rows: Converters::OrderDetailToJournalRows
+  split_accounts/split_account:
+    order_detail_to_journal_rows: SplitAccounts::Converters::OrderDetailToJournalRows
```

Comment:
I think I see now: for the NU specific overrides. One idea I had was to have a `JournalRowConverters::BaseOrderDetailConverter`, which is always abstract, and then a `DefaultOrderDetailConverter` that inherits and has just the simple implementation of `{ account: order_detail.product.account }` that gets merged in with everything else. That way there's an example of how to build your own.